content/en/docs/use-cases/gh-actions-app-promotion.md

```diff
@@ -0,0 +1,205 @@
+---
+title: GitHub Actions Basic App Promotion
```

Comment:
For this title I see very different guide where we use `flux build artifact` and `flux tag artifact` to showcase a promotion workflow for the whole app definition (container+manifests).

Could we use "GitHub Actions Basic App Builder" as the title instead?